lib/vaahextendflutter/services/logging_library/logging_library.dart

```diff
@@ -80,24 +82,24 @@ class Log {
       Console.danger('$throwable\n$hint', data);
     }
     if (_config.enableCloudLogs && !disableCloudLogging) {
-      final hintWithData = {
-        'hint': hint,
-        'data': data,
+      final hintWithData = <String, Object>{
+        'hint': hint ?? {},
+        'data': data ?? {},
```

Comment:
@we-khushal-y001 to remove logging service changes from this PR.

Reply:
Please resolve